### PR TITLE
Set the statement timeout for each statement.

### PR DESCRIPTION
**Pull request by geekbeast**

### CODE REVIEW

File: src/main/java/com/openlattice/postgres/streams/BasePostgresIterable.kt [hunk 1]
```diff
@@ -71,11 +73,18 @@ open class StatementHolderSupplier(
         return connection.createStatement()
     }
 
+    open fun setStatementTimeout(statement: Statement): Statement {
+        if(statementTimeoutMillis > 0 ) {
+            statement.execute("SET statement_timeout = '${statementTimeoutMillis}';")
```
Comment (UnsungHero97) [inline src/main/java/com/openlattice/postgres/streams/BasePostgresIterable.kt:78]: Do you need the single quotes?

File: src/main/java/com/openlattice/postgres/streams/BasePostgresIterable.kt [hunk 2]
```diff
@@ -14,6 +14,7 @@ import java.sql.SQLException
 import java.sql.Statement
 import java.util.*
 import java.util.concurrent.Executors
+import java.util.concurrent.TimeUnit
```
Comment (UnsungHero97) [inline src/main/java/com/openlattice/postgres/streams/BasePostgresIterable.kt:17]: This looks like an unused import.

**Review by UnsungHero97 — APPROVED** [review]

File: src/main/java/com/openlattice/postgres/streams/BasePostgresIterable.kt (hunk 1 above)
Comment (geekbeast) [reply src/main/java/com/openlattice/postgres/streams/BasePostgresIterable.kt:78]: yes, but I forgot to add ms

File: src/main/java/com/openlattice/postgres/streams/BasePostgresIterable.kt (hunk 2 above)
Comment (geekbeast) [reply src/main/java/com/openlattice/postgres/streams/BasePostgresIterable.kt:17]: good catch

**Review by UnsungHero97 — APPROVED** [review]

**Review by anzioka — APPROVED** [review]

**Pull request merged**